quickstarts/jumpstart/alerts/outlier-alert.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+# Name of the alert
+name: Name of this alert condition
+
+# Description and details
+details: |+
+  Describe the purpose of this alert condition.
+  You can use multiple lines.
+  Try to be as descriptive as possible and provide "The Why".
+
+# Type of alert
+type: OUTLIER
```

Comment:
type: either needs to be STATIC or BASELINE. Also I suggest renaming this alert name as outlier alerts have been deprecated. 

SEE this PR for more info: https://github.com/newrelic/newrelic-quickstarts/pull/648